documentation/modules/ROOT/pages/security-oidc.adoc

```diff
@@ -0,0 +1,188 @@
+= Security using OpenID Connect
+
+== Securing endpoints using Bearer Token Authorization
+
+You can protect your JAX-RS microservices by using Bearer Token Authorization where Bearer Tokens are issued by OpenId Connect and OAuth 2.0 compliant Authorization Servers such as https://www.keycloak.org/about.html[Keycloak].
+
+Quarkus introduced an experimental *Dev Services For Keycloak* feature which is enabled by default when the `quarkus-oidc` extension is started in dev mode. It starts a Keycloak container and initializes it by registering the existing Keycloak realm or creating a new realm with the client and users for you to start developing your Quarkus application secured by Keycloak immediately.
+
+So let's see how you can start using Bearer Tokens for securing your endpoints.
+
+== Add the OIDC extension
+
+Just open a new terminal window, and make sure you’re at the root of your `{project-name}` project, then run:
+
+[.console-input]
+[source,bash,subs="+macros,+attributes"]
+----
+./mvnw quarkus:add-extension -Dextensions="io.quarkus:quarkus-oidc"
+----
+
+== KeyCloak setup
+Go to http://localhost:8080/q/dev/[Dev UI] and select the OpenId Connect Card linking to a Keycloak page. Click on the http://localhost:8080/q/dev/io.quarkus.quarkus-oidc/provider[Provider: Keycloak link] and you will see a Keycloak page which will be presented slightly differently depending on how Dev Services for Keycloak feature has been configured.
```

Comment:
Add a new line between section and paragraph.

Reply:
Great catch! Thank you!

Reply:
Fixed with the latest commit.